lib/move-sync/index.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+// most of this code was written by Andrew Kelley
+// licensed under the BSD license: see
+// https://github.com/andrewrk/node-mv/blob/master/package.json
+// This is the sync version that somehow follows the same pattern.
+
+const fs = require('graceful-fs')
+const path = require('path')
+const copySync = require('../copy-sync').copySync
+const removeSync = require('../remove').removeSync
+const mkdirpSync = require('../mkdirs').mkdirsSync
+
+function moveSync (src, dest, options) {
+  options = options || {}
+  const overwrite = options.overwrite || options.clobber || false
+
+  if (path.resolve(src) === path.resolve(dest)) return
```

Comment:
When src and dest are the same, currently it just `return`. Which one is better, **return** or **throw an error**?